pandas/core/nanops.py

```diff
@@ -1332,31 +1332,34 @@ def nancorr(
 
 
 def get_corr_func(method):
-    if method in ["kendall", "spearman"]:
-        from scipy.stats import kendalltau, spearmanr
-    elif method in ["pearson"]:
-        pass
+    if method == "kendall":
+        from scipy.stats import kendalltau
+
+        def func(a, b):
+            return kendalltau(a, b)[0]
+
+        return func
```

Comment:
i would make all of these lambda, e.g.(for all 3 cases)
```
return lambda a, b: kendaltau(a, b)[0]
```

Reply:
hah, i asked for these to be changed from lambda to non-lambdas for nicer tracebacks

Reply:
oh really it does? hmm, ok, then that's fine